modules/installation-generate-aws-user-infra-install-config.adoc

```diff
@@ -116,6 +123,22 @@ publish: Internal
 By setting this option, you create an internal Ingress Controller and a private load balancer.
 endif::restricted[]
 
+ifdef::localzone[]
+. Edit the `install-config.yaml` file to provide the subnets for the availability zones that your VPC uses:
++
+[source,yaml]
+----
+compute:
+platform:
+  aws:
+    subnets: <1>
+    - subnet-1
+    - subnet-2
+    - subnet-3
+----
+<1> Add the `subnets` section and specify the `PublicSubnetIds` from the output of the CloudFormation template for the VPC.
```

Comment:
This is one of the most important piece of the documentation, this will be the unique part when installing in existing VPC, and here should be have both subnetIDs for **both** CloudFormation executions. So I would improve this example to keep closer to the real values.
Let me take some suggestions of what could be, feel free to adapt:

```suggestion
    subnets: <1>
    - publicSubnetId-1  <2>
    - publicSubnetId-2
    - publicSubnetId-3
    - privateSubnetId-1
    - privateSubnetId-2
    - privateSubnetId-3
    - publicLocalZoneSubnetId-1  <3>
----
<1> Add the `subnets` section and specify the `PrivateSubnetIds`, and `PublicSubnetIds` from the outputs of the CloudFormation templates for the VPC and Local Zones.
```